src/webgpu/shader/execution/expression/call/builtin/atomics/atomicCompareExchangeWeak.spec.ts

```diff
@@ -339,6 +339,11 @@ struct __atomic_compare_exchange_result<T> {
     const numInvocations = t.params.workgroupSize;
     const scalarType = t.params.scalarType;
 
+    t.skipIf(
```

Comment:
nit: maybe filter out the params instead of skipping in test body?

Reply:
Thanks for the suggestion. I don't think I  can filter these out before without a device since I'm comparing to `device.limits.maxComputeWorkgroupSizeX`.  I could hard code those limits somewhere but it seems better to look them up in the one place that would be correct in the future.

Reply:
Np for leaving as is then